ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/peertask/task/GetReceiptsFromPeerTask.java

```diff
@@ -132,4 +132,8 @@ public Predicate<EthPeer> getPeerRequirementFilter() {
   public boolean isSuccess(final Map<BlockHeader, List<TransactionReceipt>> result) {
     return !result.isEmpty();
   }
+
+  public Collection<BlockHeader> getBlockHeaders() {
```

Comment:
I can, but it means using reflection to access the fields in tests. It ends up looking pretty ugly, but it can be done. Let me know if you think it's worthwhile.